BASE/src/Microsoft.ApplicationInsights/Metrics/Implementation/AutocollectedMetricsExtraction/ExceptionMetricsExtractor.cs

```diff
@@ -0,0 +1,153 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Metrics;
+    using static System.FormattableString;
+
+    /// <summary>
+    /// An instance of this class is contained within the <see cref="AutocollectedMetricsExtractor"/> telemetry processor.
+    /// It extracts auto-collected, pre-aggregated (aka. "standard") metrics from ExceptionTelemetry objects which represent
+    /// invocations of the monitored service.
```

Comment:
pre-aggregated != standard metric. Any metric can be pre-aggregated.
The particular class is extracting pre-agg metric called Exception/count, which is defined as a StandardMetric in application insights.

Reply:
Fixed